vql/parsers/plist.go

```diff
@@ -2,25 +2,47 @@ package parsers
 
 import (
 	"context"
-
 	"github.com/Velocidex/ordereddict"
-	"howett.net/plist"
+        "howett.net/plist"
 	"www.velocidex.com/golang/velociraptor/glob"
+	utils "www.velocidex.com/golang/velociraptor/utils"
 	vql_subsystem "www.velocidex.com/golang/velociraptor/vql"
 	vfilter "www.velocidex.com/golang/vfilter"
 )
 
-type _PlistParserArgs struct {
+/*
+   Velociraptor - Hunting Evil
+   Copyright (C) 2019-2021 Velocidex Innovations.
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Affero General Public License as published
+   by the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU Affero General Public License for more details.
+   You should have received a copy of the GNU Affero General Public License
+   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+type _PlistPluginArgs struct {
+	Filenames []string `vfilter:"required,field=file,doc=A list of files to parse."`
+	Accessor string `vfilter:"optional,field=accessor,doc=The accessor to use."`
+}
+
+type _PlistFunctionArgs struct {
 	Filename string `vfilter:"required,field=file,doc=A list of files to parse."`
 	Accessor string `vfilter:"optional,field=accessor,doc=The accessor to use."`
 }
 
-type PlistParser struct{}
+type PlistFunction struct{}
+
+type _PlistPlugin struct{}
 
-func (self *PlistParser) Call(ctx context.Context,
+func (self *PlistFunction) Call(ctx context.Context,
```

Comment:
Can you please rearrange so the methods go together with the object? it is a bit easier to follow when all related methods are together.

Reply:
I've re-arranged it a bit and added another commit.  Let me know if that looks in line with what you are expecting.